calico/getting-started/windows-calico/quickstart.md

```diff
@@ -376,6 +377,83 @@ The following steps install a Kubernetes cluster on a single Windows node, with
 %>
 {% endtabs %}
 
+Congratulations! You now have a Kubernetes cluster with {{site.prodnameWindows}} and a Linux control node.
+
+#### Install {{site.prodnameWindows}} using HostProcess containers
+
+With Kubernetes v1.22, a new Windows container type called "HostProcess containers" can run directly on the host with access to the host network namespace,
+storage and devices. With this feature, {{site.prodnameWindows}} can now be installed and managed using Kubernetes resources such as Daemonsets and ConfigMaps,
+instead of needing to configure and install {{site.prodnameWindows}} manually on each node. Using this installation method, the {{site.prodnameWindows}}
+services are no longer registered on the host. Instead, the services are run directly within HostProcess containers.
+
+> **Note**: This installation method is a tech preview and should not be used for production clusters. Upgrades from a tech preview version of this
```

Comment:
I would make this a WARNING! or Important! or someone may miss it. And I would move it directly after the header,
#### Install {{site.prodnameWindows}} using HostProcess containers. 

Actually, I wonder if we should put this method as a separate file with the standard "tech preview" banner until it is ready. Why would PM want anyone to try this if upgrade would not be seamless? Do we know customers who are ready to take on the burden of not having a seamless upgrade? If the idea is just to show we have this feature, wonder if separate file is better until it becomes GA.

Reply:
I'll move the note under the header, I think that's a good idea.

The quickstart is just to give users a quick demo install, with options to try a new installation method. I think it's fair that a tech preview does not guarantee seamless upgrade. After all, the feature may change before it becomes GA.